src/app/users/customer/booking/first-page-booking/first-page-booking.component.ts

```diff
@@ -25,16 +36,63 @@ export class FirstPageBookingComponent implements OnInit {
     this.currentScreen = screenNumber;
   }
 
-  constructor(private http: HttpClient) {}
+  constructor(private http: HttpClient, 
+    private loginService: LoginService, 
+    private flightSelectionService: FlightSelectionService,
+    private router: Router
+  ) {}
 
   ngOnInit() {
-    this.fetchCustomerData();
+    this.fetchAirports();
   }
 
-  fetchCustomerData() {
-    const customerId = 'ad237302-4b0c-48bf-abd9-d01a0c6e8a2e'; // Replace with actual customer ID
-    this.http.get<any>(`http://localhost:3000/customers/${customerId}`).subscribe(data => {
-      this.milesBalance = data.miles;
+  fetchAirports() {
+    this.http.get<any>('http://localhost:3000/flights/airports').subscribe(data => {
+      console.log("Dados recebidos dos aeroportos:", data.airports);
+      this.listaAeroportos = data.airports
+    }, error => {
+      console.error("Erro ao buscar aeroportos:", error);
+    });
+  }
+  
+  searchFlights() {
+    console.log(this.flight)
+    this.http.post<any>('http://localhost:3000/flights/travels', this.flight).subscribe(data => {
+      console.log("Voos recebidos:", data.travels);
+          // Filtra os voos para incluir apenas da data atual para frente
+      const now = new Date();
+      this.listaVoos = data.travels.filter((travel: Flight) => {
+        if (travel.dataVoo) {
+          const dataVoo: Date = new Date(travel.dataVoo);
+          return (
+            dataVoo.getFullYear() > now.getFullYear() ||
+            (dataVoo.getFullYear() === now.getFullYear() && dataVoo.getMonth() > now.getMonth()) ||
+            (dataVoo.getFullYear() === now.getFullYear() && dataVoo.getMonth() === now.getMonth() && dataVoo.getDate() >= now.getDate())
+          );
+        }
+        return false
```

Comment:
don't do this, that's confusing its better to just turn into a timestamp andcompare like this:

```
const nowTimestamp = now.getTime();
const dataVooTimestamp = dataVoo.getTime();
return dataVooTimestamp >= nowTimestamp;
```

Reply:
okay, i gonna fix it